### ISSUE TITLE
Setup model permissions

### ISSUE THREAD

**Issue by justino599** [Teamable-Analytics/teamable-backend#6]

**Comment by Opeyem1a** [comment]
wdym by this issue? Permissions as in DB constraints?

If you mean API permissions, this is being left intentionally undone

**Comment by justino599** [comment]
Yeah, I mean API permissions, I think right now any unauthenticated entity would have full read/write access to the db through the API. Is it being left undone because the API won't be publicly exposed?
